constellations/gemini-admin/src/utils/inputSelect.js

```diff
@@ -0,0 +1,105 @@
+"use strict";
+require("./inputText.sass");
+import React from "react";
+import Checkbox from "../../assets/checkmark_green.svg";
+
+/*
+ * Props for the InputSelect Component:
+ *
+ * value: Current selected option
+ *
+ * onChangeCallback: Function that is called when the selection changes.
+ *
+ * required: Whether or not the selection  is required (omit if false).
+ *
+ * hasNoDefault: If true, an extra option -- Select an option -- will be added as the default.
+ *                  It cannot be reselected after another option is chosen.
+ *
```

Comment:
sorry can you add one more (optional) field.

If the list is empty, but it really shouldn't be. Something went wrong or a user action is required. For example, if I'm creating a class, but there's no program or semester to choose from. We should ask the user to first add a program/semester.

In this case, add another field `errorMessageIfEmpty` which takes in a string. If the list is empty, display this message. If the list is non-empty, do nothing.